integration-tests/gradle/src/test/resources/jandex-basic-multi-module-project/application/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
Can you remove this file please? It's not needed

Reply:
Yes sir

Reply:
Well, that's the reason it fails :) This is the first page it checks to make sure it started. Otherwise, you need to be checking the `hello` endpoint.

Reply:
Better to check the `hello` endpoint I would say instead of this useless file :)